tests/end_to_end/serving.py

```diff
@@ -15,22 +15,34 @@
 SLEEP_DURATION = 30
 NUMBER_OF_SLEEPS = 20
 
-def serve_data():
-    client = ff.ServingClient()
 
+client = ff.ServingClient()
+
+def serve_data():
     for _ in range(NUMBER_OF_SLEEPS):
         try: 
             dataset = client.training_set(f"fraud_training_{VERSION}", "quickstart")
             training_dataset = dataset.repeat(10).shuffle(1000).batch(8)
             for i, feature_batch in enumerate(training_dataset):
-                print(feature_batch.to_list())
-                if i >= 5:
+                if i >= 1:
                     return
+                print(feature_batch.to_list())
+                
         except Exception as e:
             print(f"\twaiting for {SLEEP_DURATION} seconds")
             time.sleep(SLEEP_DURATION)
     
     raise Exception(f"Serving for {VERSION} could not be completed.")
```

Comment:
version not picking up, do formatting strings need single quotes?

Reply:
you can use single or double quotes. The issue is that Python sets it as environment variable but it isn't available outside of the script.